### ISSUE TITLE
Paxman Safari Render issue on Macbook

### ISSUE THREAD

**Issue by ankurgoel11** [raphamorim/origami.js#69]
I found that Pacman is not getting rendered on the safari browser.

http://origamijs.com/docs/

On the above link, the pacman is visible on all browser except safari (On Macbook Pro)

**Comment by raphamorim** [comment]
Hi @ankurgoel11, thanks for reporting it. 
What's your Safari version?

**Comment by ankurgoel11** [comment]
Version 10.1.2 (10603.3.8)

**Comment by squidster3128** [comment]
Yeah, I am having the same problem with the latest Safari from High Sierra!

**Comment by squidster3128** [comment]
Any progress on this?
